setup.py

```diff
@@ -39,14 +39,15 @@ def get_version(*file_paths):
 setup(
     name='hashID',
     packages=find_packages(exclude=['HASHINFO.xlsx']),
```

Comment:
I made a mistake here excluding `HASHINFO.xlsx`, this is not needed at all - the line should be just
`packages=find_packages(),`